.github/workflows/build-images-action.yml

```diff
@@ -23,6 +23,19 @@ jobs:
       QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
       QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  build_ironic_cs10:
+    name: Build Ironic container image based on CentOS Stream 10
+    env:
+      IMAGE_BUILD_ARGS: ['BASE_IMAGE=quay.io/centos/centos:stream10']
+    if: github.repository == 'metal3-io/ironic-image'
+    uses: metal3-io/project-infra/.github/workflows/container-image-build.yml@main
+    with:
+      image-name: 'ironic_cs10'
```

Comment:
This needs to be created in Quay before this merges, otherwise the push will fail.

Reply:
created https://quay.io/repository/metal3-io/ironic_cs10